src/components/picker/nimble-picker.js

```diff
@@ -14,6 +14,8 @@ import { Anchors, Category, Preview, Search } from '..'
 const I18N = {
   search: 'Search',
   notfound: 'No Emoji Found',
+  skinhover: 'Select Skin Tone',
+  skintext: 'Choose your default skin tone ',
```

Comment:
Is this the copy we'll want to use in Trello?

Reply:
Yes, this content was taken from the card!